test/services/api/handlers/users.js

```diff
@@ -0,0 +1,414 @@
+var Lab = require('lab'),
+  lab = exports.lab = Lab.script(),
+  experiment = lab.experiment,
+  before = lab.before,
+  after = lab.after,
+  test = lab.test,
+  expect = require('code').expect,
+  server;
+
+before(function(done) {
+  require('../../../mocks/server')(function(obj) {
+    server = obj;
+    done();
+  });
+});
+
+after(function(done) {
+  server.stop(done);
+});
+
+experiment('User Handler', function() {
+  experiment('Create', function() {
+    test('Creates a new user', function(done) {
```

Comment:
Lots of boilerplate in these tests. Curious what you think about trimming these down a bit by creating another test fixture.